apps/checkout-app/backend/client.ts

```diff
@@ -9,6 +9,7 @@ interface ClientParams {
 
 export const getClient = (params: ClientParams = {}) => {
   const { apiUrl = envVars.apiUrl, appToken = getAuthToken() } = params;
+  console.log(apiUrl, appToken);
```

Comment:
Question - what is our approach to logs? Do we use console logs now only? Do we have plan for some more sophisticated logger?